src/server/models/users.model.js

```diff
@@ -2,13 +2,12 @@
 
 import mongoose from 'mongoose';
 
-/**
+/*
  * Define the schema model for user.
  */
 
 const Schema = mongoose.Schema;
 let userSchema = new Schema({
-  userId: String,
```

Comment:
Why was this deleted?? Even though email address is enforced to be unique, user ID (numeric) should be there.